### PR TITLE
VAGOV-6064: Truer wysiwyg.

### PR DESCRIPTION
**Pull request by kevwalsh**
https://va-gov.atlassian.net/browse/VAGOV-6064
* Create a page
* Add a wysiwyg field in the content block
* Try out every HTML element, paragraph format, and block style, including link buttons. 
* They should approximate what the VA Design system

* Save the node
* On the node view page, everything in `div.node__content`  should look more like the front end.

### CODE REVIEW

File: docroot/themes/custom/vagovadmin/vagovadmin.info.yml [hunk 1]
```diff
@@ -3,6 +3,10 @@ type: theme
 description: 'Overrides for the Seven theme.'
 core: 8.x
 base theme: seven
+ckeditor_stylesheets:
+- https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700&display=swap
+- https://fonts.googleapis.com/css?family=Bitter:400,700&display=swap&subset=latin-ext
+- css/wysiwyg.css
```
Comment (swirtSJW) [inline docroot/themes/custom/vagovadmin/vagovadmin.info.yml]: @kevwalsh Does `css/admin-styles.css` need to be in here too?

Comment (kevwalsh) [reply docroot/themes/custom/vagovadmin/vagovadmin.info.yml]: No, it's in 

```libraries:
  - vagovadmin/admin-styles```

**Comment by ElijahLynn** [comment]
This failed due to comment length exceeding maximum for GitHub. 
image

**Comment by ElijahLynn** [comment]
> This failed due to comment length exceeding maximum for GitHub.
> image
> 
> Working on this.

Oh, actually it failed because of 
>       Current response status code is 404, but 200 expected. (Behat\Mink\Exception\ExpectationException)

**Comment by ElijahLynn** [comment]
http://pr606.ci.cms.va.gov/static/health-care/about-va-health-benefits/vision-care/blind-low-vision-rehab-services Is not actually there. So it is possible something legit came up.

**Comment by ElijahLynn** [comment]
The new commit to comment that out was not actually deployed it appears. 
http://devshop.cms.va.gov/node/22284

**Comment by ElijahLynn** [comment]
The CI ENV for this PR had it on the wrong branch for some reason. Closing to destroy and reopen to build anew.

**Comment by ElijahLynn** [comment]
I ran tests manually on the ENV with `composer yaml-tests --tests-file=tests.yml --ansi` and it passed. The /static/health-care/index.html was not getting generated with automation.

**Comment by ElijahLynn** [comment]
I created #608 to revert the test change we made, :crossed_fingers: that _just works_.

**Pull request merged**